index.js

```diff
@@ -2,12 +2,14 @@ const fs = require('fs');
 const pkg = require('./package.json');
 const getParser = require('./lib/parser');
 const getMappers = require('./lib/mappers');
-const { prettyJSON, writeFile } = require('./lib/utils');
+const { prettyJSON, writeFile, downloadFile } = require('./lib/utils');
 const mapToStyleDictionaryTokens = require('./lib/styleDictionary');
 
+const ASSETS_DIR = 'assets';
+
 module.exports = async (args, flags) => {
   if (flags.version) return pkg.version;
-  if (args.length <= 0) throw new Error('No file input passed after npm start');
+  if (args.length <= 0 && !flags.token) throw new Error('No file input or token passed after npm start');
```

Comment:
We can not require a token since Sketch does not use a token, only Figma does.
Checking tokens should be done in the correct parser